app/controllers/application_controller.rb

```diff
@@ -6,4 +6,8 @@ def command_injection
   def command_injection2
     system(params[:command])
   end
+
+  def command_injection3
+    system(params[:command])
```

Comment:
<article><p><b>[Brakeman]</b> <span>Possible command injection</span></p><details><p></p><p>11 line , 0 column in <code>app/controllers/application_controller.rb</code></p><dl><dt>Rule</dt><dd>Execute</dd><dt>Severity</dt><dd>High</dd></dl></details></article>